src/pages/patientView/PatientViewPage.tsx

```diff
@@ -276,9 +273,6 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
                         this.setState(({myCancerGenomeData: _result} as IPatientViewState));
                     });
 
-                    this.fetchHotspotsData(patientViewPageStore.mutationData.result).then((hotspotsData: IHotspotData) => {
```

Comment:
we need to clean up all of the legacy stuff in this file.  I think we should be able to completely do away with use of component state.